.dotnet/src/Generated/Models/AudioTranslation.Serialization.cs

```diff
@@ -26,7 +26,7 @@ void IJsonModel<AudioTranslation>.Write(Utf8JsonWriter writer, ModelReaderWriter
             writer.WritePropertyName("language"u8);
             writer.WriteStringValue(Language);
             writer.WritePropertyName("duration"u8);
```

Comment:
Unrelated to your PR, this code crashes because Duration was customized to be made nullable but this code doesn't account for the possibility of a null value... Sent a message to the generator team.